chargebee_byte/requests.py

```diff
@@ -0,0 +1,31 @@
+from chargebee_byte.parameters import generate_sorting_parameters, generate_equals_parameters, generate_comparison_parameters, \
```

Comment:
This looks pretty annoying. What do you think about `from chargebee_byte import parameters` and then renaming the functions just `def sorting(params)`, `def equals(params)` etc.? The function names will be a little bit less descriptive but generally better to read/write.

Reply:
I personally don't really prefer that. I rather be explicit when it comes to function names. In this case I especially think it's hard to see `sorting` and see/know what it does on first look. Personally I also like to do something like this when I'm writing code:
```
# I need to generate some parameters, but don't know what the function is
generate_  # ctrl + double space to get a list of all generate_ functions.
# Choose the generate_ function I need.
```
By changing the function names to something shorter it would, for me personally, make it less clearer and harder to write code faster.